ivy/functional/frontends/jax/numpy/indexing.py

```diff
@@ -101,6 +101,12 @@ def indices(dimensions, dtype=int, sparse=False):
         return ivy.stack(grid, axis=0).astype(dtype)
 
 
+@to_ivy_arrays_and_back
+def choose(arr, choices, out=None, mode="raise"):
+    """jax.numpy.choose(arr, choices, out=None, mode='raise')"""
```

Comment:
Why there is this comment here?

Reply:
because I wanted to resolve lint warnings. I will move that, I think is better